torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -130,9 +137,149 @@ def _lower_weighted_ref_module(model: QuantizedGraphModule) -> QuantizedGraphMod
         model.recompile()
     return model
 
+def _lower_quantized_binary_op(
+    model: QuantizedGraphModule,
+    qconfig_map: Dict[str, QConfigAny]
+) -> QuantizedGraphModule:
+    modules = dict(model.named_modules(remove_duplicate=False))
+
+    def get_bop_patterns(bop: Callable) -> Pattern:
+        return [
+            (torch.quantize_per_tensor,
+             (bop, "dequantize", "dequantize"),
+             MatchAllNode, MatchAllNode, MatchAllNode)
+            for bop_pattern in [
```

Comment:
This is not used. Did you mean the following?
```
(torch.quantize_per_tensor, bop_pattern, MatchAllNode, MatchAllNode, MatchAllNode)
```

Reply:
ah sorry, let me fix it